### ISSUE TITLE
Question: What's the purpose of #frame_rate in mp4mux

### ISSUE THREAD

**Issue by XingzhaoYun** [axiomatic-systems/Bento4#881]
Hi, I have a question regarding to the frame_rate for video muxing in mp4mux.

I found the parameter frame_rate was introduced many years ago, during which the vps_num_units_in_ticks and vps_time_scale were not parsed.

With the updates of mp4mux, it now can parse VPS to get the real timescale from elementary stream, so I'm wondering what's the use case for keeping this frame_rate?

Thanks!

**Comment by barbibulle** [comment]
You're right, `frame_rate` could me made optional in cases where it can be inferred from the elementary stream. I'll try to add support for that in a future release.
